### ISSUE TITLE
CLIMADA petals citation

### ISSUE THREAD

**Issue by manniepmkam** [CLIMADA-project/climada_petals#98]
Currently there is no citation guidelines or zenodo release for climada petals. Shall we make one for climada petals citation?

**Comment by chahank** [comment]
Yes, good idea! We also just need to do a zenodo release (or rather decide on whether to do one or not?)
